src/components/animated/_glow.less

```diff
@@ -1,88 +1,51 @@
-.glowBorderRule(@color) {
+.glow(@color) {
+  transition: all 0.5s ease;
```

Comment:
Put this in `.glow-btn` like it was to not generate this code many times when compiling to CSS.

Reply:
fixed it